src/pudl_archiver/cli.py

```diff
@@ -64,8 +64,8 @@ def parse_main(args=None):
     )
     (
         parser.add_argument(
-            "--download-dir",
```

Comment:
I removed the `download-dir` option because I don't think we're using it much and it could now be replaced with using the fsspec depositor with a local path to view resources locally.

I don't love that there's coupling between the `deposition-path` and `depositor` options, but I couldn't think of a better solution with spending more time than I really wanted on this.

Reply:
I don't think it's impossible, but I felt like keeping both of these args around or overloading it to have 2 different meanings would cause more confusion than it's worth. There are still lots of avenues for reviewing data before publishing to zenodo like using the sandbox, turning `auto-publish` off and manually reviewing a draft, or using the new fsspec depositor to download locally first.